test/sql/hnsw_select.sql

```diff
@@ -9,8 +9,18 @@ CREATE INDEX ON small_world USING hnsw (v) WITH (dims=3, M=5, ef=20, ef_construc
 \ir utils/sift1k_array.sql
 CREATE INDEX ON sift_base1k USING hnsw (v) WITH (dims=128, M=5, ef=20, ef_construction=20);
 
+CREATE TABLE test1 (id SERIAL, v REAL[]);
+CREATE TABLE test2 (id SERIAL, v REAL[]);
+INSERT INTO test1 (v) VALUES ('{5,3}');
+INSERT INTO test2 (v) VALUES ('{5,4}');
+CREATE INDEX ON test1 USING hnsw (v);
+
 SET enable_seqscan = false;
 
+-- Verify that basic queries still work
```

Comment:
add to comment: 
-- Verify that basic queries still work given our query parser and planner hooks

Reply:
Agree, done.